<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SegmentationOptionPage</class>
 <widget class="QWidget" name="SegmentationOptionPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SegmentationOptionPage</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>笔刷颜色</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,0">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>前景</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ObjColorBt">
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 0);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="shortcut">
           <string>Ctrl+O</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>背景</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="BkgColorBt">
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 0, 0);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="shortcut">
           <string>Ctrl+O</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>32</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>分割算法</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="MethodComboBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>32</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>GraphCut</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>GrabCut</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>LazyBrush</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mixture</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>平滑参数</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>lamda</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lamda_lineEdit">
        <property name="whatsThis">
         <string/>
        </property>
        <property name="inputMask">
         <string extracomment="alpha值"/>
        </property>
        <property name="text">
         <string>1.5</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>kappa</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="kappa_lineEdit">
        <property name="whatsThis">
         <string/>
        </property>
        <property name="inputMask">
         <string extracomment="kappa值"/>
        </property>
        <property name="text">
         <string>4.0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_13">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>80</height>
          </size>
         </property>
         <property name="title">
          <string>Edge选项</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QRadioButton" name="use_log_input">
            <property name="text">
             <string>使用LoG作为输入</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="use_canny_input">
            <property name="text">
             <string>使用Canny作为输入</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_10">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>80</height>
          </size>
         </property>
         <property name="title">
          <string>Canny选项</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QCheckBox" name="canny_checkbox">
            <property name="text">
             <string>使用Canny增强边缘</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="canny_erode_checkbox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>对Canny结果Erode</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_7">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>80</height>
          </size>
         </property>
         <property name="title">
          <string>组合方式</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>9</number>
          </property>
          <property name="leftMargin">
           <number>9</number>
          </property>
          <property name="topMargin">
           <number>9</number>
          </property>
          <property name="rightMargin">
           <number>9</number>
          </property>
          <item>
           <widget class="QRadioButton" name="linear_radioButton">
            <property name="text">
             <string>线性</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QRadioButton" name="nolinear_radioButton">
              <property name="text">
               <string>非线性</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox">
              <item>
               <property name="text">
                <string>平方</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>立方</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_8">
         <property name="title">
          <string>混合比重</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>LazyBrush算法比重</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="edge_weight_lineEdit"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="CannyGroupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>80</height>
      </size>
     </property>
     <property name="title">
      <string>Canny参数</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="spacing">
         <number>12</number>
        </property>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>canny最小阈值</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="canny_min_edit">
          <property name="whatsThis">
           <string/>
          </property>
          <property name="inputMask">
           <string extracomment="alpha值"/>
          </property>
          <property name="text">
           <string>60</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>12</number>
        </property>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>canny最大阈值</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="canny_max_edit">
          <property name="whatsThis">
           <string/>
          </property>
          <property name="inputMask">
           <string extracomment="kappa值"/>
          </property>
          <property name="text">
           <string>180</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <zorder>stackedWidget</zorder>
  <zorder>CannyGroupBox</zorder>
  <zorder>groupBox_3</zorder>
  <zorder>groupBox</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
